internal/gitinterface/common.go

```diff
@@ -49,7 +48,7 @@ func CreateTestGitRepository(t *testing.T, dir string) *Repository {
 		t.Fatal(err)
 	}
 
-	repo := &Repository{gitDirPath: path.Join(dir, ".git"), clock: testClock}
+	repo := &Repository{gitDirPath: filepath.Join(dir, ".git"), clock: testClock}
```

Comment:
I was wrong on slack. The mix of `\\` and `/` in the test is happening here, so this change is good!